server/selinux/server/pulp-server.te

```diff
@@ -7,10 +7,14 @@ require {
     type puppet_etc_t;
     type httpd_t;
     type cert_t;
+    type nfs_t;
```

Comment:
I don't think we need `nfs_t`, the user should configure their NFS with the `context` option w/ the expected labels. Then the other `:map` statement should cover it.